generator/pygen/pygen.cpp

```diff
@@ -347,6 +347,8 @@ BOOST_PYTHON_MODULE(pygen)
         .value("contact_twitter", Metadata::EType::FMD_CONTACT_TWITTER)
         .value("contact_vk", Metadata::EType::FMD_CONTACT_VK)
         .value("contact_line", Metadata::EType::FMD_CONTACT_LINE);
+        .value("destination", Metadata::EType::FMD_DESTINATION);
+        .value("destination_ref", Metadata::EType::FMD_DESTINATION_REF);
```

Comment:
Hm, Did you get what for this pygen.cpp module?
And why "destination_ref", if we should get "destination:ref"

Reply:
No, I didn't :)
I applied the same logic as for FMD_TURN_LANES_FORWARD, FMD_BUILDING_LEVELS, etc.

BTW, can't we use duplication of matching in indexer::ToString, indexer::Metadata::TypeFromString and in Pygen.
We need 2 direct and 1 reverse transformation.
E.g. create static array and initialize it as `a[Metadata::EType] = value;`
And in cycle populate `map<string, Metadata::EType>` and extend it manually where needed for reverse transformation?

Reply:
OK